contributing.rst

```diff
@@ -0,0 +1,106 @@
+Contributing to Traja
+=====================
+
+Traja is a research library. Functionality must therefore be both
+cutting-edge and reliable. Traja is part of a wider project to
+increase collaboration in research, through the adoption of
+open-source contribution models. It is our hope that traja
+remains accessible to researchers and help them do higher-quality
+research.
+
+Current status
+--------------
+
+Traja is currently undergoing active development with approximately
+60 % of features present. Significant interface changes are still
+possible, however we avoid these unless absolutely necessary.
+
+The work is currently focussed on reaching version 1.0 with feature
+completeness and 95 % test coverage.
+
+The following features are required for feature completeness:
+
+* Latent space visualisers
+   * Eigenspace-based
+   * Colour-coded to visualise evolution over time
+   * Delay coordinate embeddings
+* State-space visualisers
+* Additional encoder and decoder options in AE and VAE models
+   * MLP
+   * 1d convolution
+* Pituitary gland example dataset
+* Regression output visualisers
+* VAE GAN models
+* Additional VAE latent-space shapes
+   * Uniform
+   * A shape that works for periodic trajectories (Torus?)
+* Delay coordinate embeddings
+   * Persistent homology diagrams of the embeddings
+* Automatic code formatter
+* Tutorials
+   * Find time of day based on activity
```

Comment:
Can we make these as hyperlinks?

Reply:
Done - the third links isn't ready yet(?)